CMakeLists.txt

```diff
@@ -51,6 +51,7 @@ endif()
 find_library(SCIPLIB scip PATHS ${SCIPOPT_LIB_DIR})
 find_library(MA27LIB  ma27  PATHS ${MA27LIB_DIR})
 find_library(CPLEXLIB cplex PATHS ${CPLEX_LIB_DIR})
+find_library(GUROBILIB gurobi${GUROBI_VERSION} PATHS ${GUROBI_LIB_DIR})
```

Comment:
Where do we get `GUROBI_VERSION`?

Reply:
Gurobi library is defined as `libgurobi90.dylib` on mac. "90" means version 9.0. It is different from CPLEX library `libcplex.a`.

Reply:
The code has been tested. I will resolve the confilict.

Reply:
> The code has been tested. I will resolve the confilict.

Please see this: https://github.com/Argonne-National-Laboratory/DSP/pull/63#issuecomment-634934107

Reply:
> Gurobi library is defined as `libgurobi90.dylib` on mac. "90" means version 9.0. It is different from CPLEX library `libcplex.a`.

**WHERE** does variable `GUROBI_VERSION` set?

Reply:
`GUROBI_VERSION` is set in `UserConfig.cmake`. Just pushed this file, too.

Reply:
Please consider to use this: https://support.gurobi.com/hc/en-us/articles/360039499751-CMake-C-C-compilation-of-Gurobi-projects

Reply:
> Please consider to use this: https://support.gurobi.com/hc/en-us/articles/360039499751-CMake-C-C-compilation-of-Gurobi-projects

This is the file I refer to. In Gurobi, the library files are defined [here](https://www.gurobi.com/documentation/9.0/quickstart_mac/file_overview.html). `libgurobi90.so` and `libgurobi90.dylib` are libraries for C interface on Windows and Mac, respectively.

In the `FindGUROBI.cmake` file in the quoted link, they have the following command to find the C library

```
find_library(GUROBI_LIBRARY
    NAMES gurobi gurobi81 gurobi90
    HINTS ${GUROBI_DIR} $ENV{GUROBI_HOME}
    PATH_SUFFIXES lib)
```
which is the same as what I did.

Reply:
Would not it be better to list names `gurobi` `gurobi81` `gurobi90` as in their example?

Reply:
I can also do that, but we may need to update this line in CMake files every time when Gurobi has new versions. I think the way I did is more general.

Reply:
I prefer updating cmake file over asking users to put extra information. In the sense, gurobi example is more user friendly.

Reply:
Sure. Then I will update the CMAKE file.